ansible_bender/__init__.py

```diff
@@ -6,9 +6,13 @@
 $ ab build ./playbook.yaml fedora:28 my-custom-image
 
 """
-from pkg_resources import get_distribution, DistributionNotFound
 try:
-    __version__ = get_distribution(__name__).version
-except DistributionNotFound:
+    from importlib.metadata import version, PackageNotFoundError
+except ImportError:
+    from importlib_metadata import version, PackageNotFoundError
```

Comment:
It took me probably 5 minutes to notice the 1 character difference b/w lines 10 and 12 :D